client/components/jetpack/threat-item/index.tsx

```diff
@@ -88,13 +88,24 @@ const ThreatItem: React.FC< Props > = ( {
 		if ( ! threat.fixable ) {
 			return (
 				<>
-					<p className="threat-description__section-text">
-						{ translate(
-							'Jetpack Scan cannot automatically fix this threat. We suggest that you resolve the threat manually: ' +
-								'ensure that WordPress, your theme, and all of your plugins are up to date, and remove ' +
-								'the offending code, theme, or plugin from your site.'
-						) }
-					</p>
+					{ ! threat.rows && (
+						<p className="threat-description__section-text">
+							{ translate(
+								'Jetpack Scan cannot automatically fix this threat. We suggest that you resolve the threat manually: ' +
+									'ensure that WordPress, your theme, and all of your plugins are up to date, and remove ' +
+									'the offending code, theme, or plugin from your site.'
+							) }
+						</p>
+					) }
+					{ threat.rows && (
+						<p className="threat-description__section-text">
+							{ translate(
+								'Jetpack Scan cannot automatically fix this threat. We suggest that you resolve the threat manually: ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Jetpack Scan cannot automatically fix this threat. We suggest that you resolve the threat manually: ensure that WordPress, your theme, and all of your plugins are up to date, and remove the offending code, theme, or plugin from your site.' )` *ES Score: 10*